src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { tabs } from '../../constants';
+
+export const Tabs = ({ onTabSelect }) => {
+  const initialTab = tabs.find(tab => tab.title === 'Tab 1');
+  const [activeTab, setActiveTab] = useState(initialTab.id);
```

Comment:
Pay attention to this requirement:
![image](https://github.com/mate-academy/react_tabs-js/assets/25587299/b40f4aee-b93a-420c-9353-05c3b0d42df7)